apis/python/tests/test_experiment_query.py

```diff
@@ -83,6 +84,10 @@ def soma_experiment(
 
     return _factory.open((tmp_path / "exp").as_posix())
 
+def get_soma_experiment_with_context(soma_experiment, context):
```

Comment:
Not a huge fan of this, but parametrizing the `context` in the `soma_experiment` fixture seemed even worse. Open to ideas on how to improve this.